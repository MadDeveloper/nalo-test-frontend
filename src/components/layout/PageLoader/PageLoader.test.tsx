import { BrowserRouter } from "react-router-dom"
import renderer from "react-test-renderer"
import { PageLoader } from "."

describe("Components > PageLoader", () => {
  it("should match snapshot", () => {
    const tree = renderer
      .create(
        <BrowserRouter>
          <PageLoader />
        </BrowserRouter>
      )
      .toJSON()

    expect(tree).toMatchSnapshot()
  })
})
