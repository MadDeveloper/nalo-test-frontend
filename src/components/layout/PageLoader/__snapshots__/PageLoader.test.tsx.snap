// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Components > PageLoader should match snapshot 1`] = `
<main
  className="root"
  style={
    Object {
      "opacity": 0,
    }
  }
>
  <svg
    className="root loader"
    viewBox="0 0 50 50"
  >
    <circle
      className="path"
      cx="25"
      cy="25"
      fill="none"
      r="20"
      strokeWidth="5"
    />
  </svg>
</main>
`;
